src/components/ServiceDetails/AboutService/DemoExample/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import { withStyles } from "@material-ui/styles";
+
+import ProgressBar from "../../../common/ProgressBar";
+import { useStyles } from "./styles";
+import ServiceProvider from "./ServiceProvider";
+
+class DemoExample extends Component {
+  state = {
+    activeSection: 1,
+    error: "error state message",
+    progressText: ["Configure", "Purchase", "Results"],
+    protoSpec: undefined,
+    serviceSpecJSON: undefined,
+    grpcResponse: undefined,
+  };
+
+  render() {
+    const { classes, service } = this.props;
+    const { activeSection, progressText } = this.state;
+    return (
+      <div className={classes.demoExampleContainer}>
+        <h4>Process</h4>
+        <ProgressBar activeSection={activeSection} progressText={progressText} />
+        <p>{this.props.tutorial}</p>
+        <ServiceProvider service_id={service.service_id} org_id={service.org_id} />
+      </div>
+    );
+  }
+}
+
+DemoExample.defaultProps = {
+  tutorial: `Transfer the style of a “style Image” to a “content image” by choosing them in the boxes below. You can upload
```

Comment:
Extract this into a constant.

Reply:
removed the [text](https://github.com/singnet/refactored-dapp/pull/13/commits/ff31558b08c2d932f1e21bad426a4a7b98372297)